.github/workflows/test.yaml

```diff
@@ -42,3 +42,18 @@ jobs:
 
       - name: test
         run: cargo test --verbose
+
+  wasm:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: wasm32-unknown-unknown
+
+      - name: Check
+        run: cargo check --target wasm32-unknown-unknown
```

Comment:
If this is just checking that wasm builds, could you incorporate this into the `build` job instead? E.g. use `dtolnay/rust-toolchain@stable` and put `cargo check --target wasm32-unknown-unknown` into the `build` step. Also worth updating the lint job to use `dtolnay/rust-toolchain@stable`. Otherwise, if you want to make a standalone wasm example using e.g. nodejs that makes a request, then it could have it's own job

Reply:
Ok, made some changes, added `wasm check` step under the `build` job.